packages/react/label/src/Label.tsx

```diff
@@ -94,16 +94,20 @@ const useLabelContext = (element?: HTMLElement | null) => {
   return context?.id;
 };
 
-function addLabelClickEventListener(label: HTMLSpanElement, element: HTMLElement) {
+function addLabelClickEventListener(label: HTMLSpanElement, control: HTMLElement) {
   const handleClick = (event: MouseEvent) => {
+    const isClickingControl = control.contains(event.target as Node);
+    // Ensure event was generated by a user action
+    // https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted
+    const isUserClick = event.isTrusted === true;
```

Comment:
so AFAICT this is the only real addition right?
In this case, how does this case help us?
I would have thought that the check for whether we're already clicking on the control itself would be enough.

Reply:
Is it specifically so we don't catch the one that is dispatched from `BubbleInput`?

Reply:
Yep, we only want to make sure that when it's a user clicking the label, we click the button for them.